podman_tutorial.md

```diff
@@ -0,0 +1,153 @@
+![PODMAN logo](https://cdn.rawgit.com/kubernetes-incubator/cri-o/master/logo/crio-logo.svg)
+
+# Basic Setup and Use of Podman
+Podman is a utility provided as part of the libpod library.  It can be used to create and maintain
+containers. The following tutorial will teach you how to set up podman and perform some basic
+commands with podman.
+
+## Install podman on Fedora
+Many of the basic components to run podman are readily available from the Fedora RPM repositories; the only
+exception is podman itself.  In this section, we will help you install all the runtime and build dependencies
```

Comment:
Podman not having an rpm is not going to be the case forever right?  I think I'd note that and say this is a way to get Podman up until a rpm can be made available on Fedora.

Reply:
done